### PR TITLE
2023.6.0

### PR DESCRIPTION
**Pull request by frenck**
⚠️ Please don't merge this PR, it is here for review purposes.

### CODE REVIEW

File: source/_posts/2023-06-07-release-20236.markdown [hunk 1]
```diff
@@ -0,0 +1,867 @@
...
+
+I am excited about so many things being shipped in this release that it is
+hard to pick a favorite 😅 This release has it all.
+
+Being able to directly use and back up my Home Assistant configuration to my
+network storage is absolutely incredible! But I also really enjoy the new and
+clean integrations dashboard and pages.
+
+The feature that greatly improved the Home Approval Factor in my household
+is definitely the addition of the new favorite colors they can quickly pick
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]: I had to read this sentence a couple of times before I got it. I don't come up with a better yet though.

File: source/_posts/2023-06-07-release-20236.markdown [hunk 2]
```diff
@@ -0,0 +1,867 @@
...
+integration dashboard. For example, if you use [ESPHome] with many devices,
+you are undoubtedly familiar with the annoying scrolling in that little card. Or,
+big error messages stretching those cards out, messing up the whole page.
+
+This release brings in a whole new integrations dashboard that addresses many
+of these issues, aiming to provide a cleaner and more consistent experience.
+
+<img class="no-shadow" src='/images/blog/2023-06/integrations-dashboard.png' alt='Screenshot showing the look and feel of the new integrations dashboard.'>
+
+Device base integrations, like ESPHome, are combined: no more scrolling.
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
Device based integrations, like ESPHome, are combined: no more scrolling.
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 3]
```diff
@@ -0,0 +1,867 @@
...
+A failing integration does not stretch the layout and no longer makes the
+layout "jump" during retries. Custom integrations and integrations relying
+on the cloud are now more visibly marked.
+
+Selecting the cog icon on the integration card will bring you to a brand new
+integrations page, showing all information about integration:
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
integrations page, showing all information about the integration:
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 4]
```diff
@@ -0,0 +1,867 @@
...
+that! On the {% my blueprints title="**Settings** > **Automations & Scenes** > **Blueprints**" %}
+page, select the three dotted menu on any blueprint to discover which
+automations are using that blueprint.
+
+Not only that, if you accidentally remove a blueprint that is still in use,
+Home Assistant will stop you from removing it and tell you which automations
+still rely on it.
+
+Hold on; there is more related! The related items in the entity information
+dialog got improved as well. Told old list of links is gone, and it now shows
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
dialog got improved as well. The old list of links is gone, and it now shows
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 5]
```diff
@@ -0,0 +1,867 @@
...
+## Copy 'n paste support in the automation editor
+
+An amazing contribution by [@karwosts] in this release that adds cut, copy
+& paste capabilities to the automation and script editors!
+
+You can now copy and paste any trigger, condition, and action into different
+places of your automation and even paste it into a different automation
+altogether!
+
+<img class="no-shadow" src='/images/blog/2023-06/cut-copy-past-automations.gif' alt='Screen recording demonstrating cutting & pasting an condition in the automation editor.'>
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2023-06/cut-copy-past-automations.gif' alt='Screen recording demonstrating cutting & pasting a condition in the automation editor.'>
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 6]
```diff
@@ -0,0 +1,867 @@
...
+
+{% details "IBM Watson TTS" %}
+
+Home Assistant was still using the legacy Python pip package resolver, which
+has been removed. Due to upstream package conflicts, the IBM Watson TTS
+integration has been disabled.
+
+The upstream project providing the libraries used has been notified repeatedly
+in, at least, the past two years; unfortunately, without response. We had no
+other choice to disable disabling this integration.
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
other choice than to disable this integration.
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 7]
```diff
@@ -0,0 +1,867 @@
...
+
+[@frenck]: https://github.com/frenck
+[#91492]: https://github.com/home-assistant/core/pull/92124
+
+{% enddetails %}
+
+{% details "IHC Controller" %}
+
+The default unit of measurement for manually set up sensors from IHC has been
+removed. Home Assistant now assume all sensors with a unit of measurement set
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
removed. Home Assistant now assumes all sensors with a unit of measurement set
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 8]
```diff
@@ -0,0 +1,867 @@
...
+
+[@Drafteed]: https://github.com/Drafteed
+[#93543]: https://github.com/home-assistant/core/pull/93543
+
+{% enddetails %}
+
+{% details "Meteorologisk institutt (Met.no)" %}
+
+The weather forecast provided by the weather entity has changed. The current
+day has been added to the forecast. If your automation or scripts relied
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
day has been added to the forecast. If your automations or scripts rely
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 9]
```diff
@@ -0,0 +1,867 @@
...
+on the first day to be tomorrow, you would need to adjust.
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
on the first day to be tomorrow, you need to adjust them.
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 10]
```diff
@@ -0,0 +1,867 @@
...
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#93549]: https://github.com/home-assistant/core/pull/93549
+
+{% enddetails %}
+
+{% details "Persistent notifications" %}
+
+Persistent notifications are no longer stored in the state machine. This means
+that we no longer create an entity for each persistent notification.
```
Comment (MartinHjelmare) [inline source/_posts/2023-06-07-release-20236.markdown:687]: Have we decided if we're going to add a template helper?

Comment (frenck) [reply source/_posts/2023-06-07-release-20236.markdown:687]: @MartinHjelmare They can do nothing. There is nothing; the template helper is in draft, not this release.

I did push a small change in https://github.com/home-assistant/home-assistant.io/pull/27692/commits/eb36ba1a21be270b872de6ce64c68797360f467b to clarify it cannot be used anymore.

File: source/_posts/2023-06-07-release-20236.markdown (hunk 1 above)
Comment (frenck) [reply source/_posts/2023-06-07-release-20236.markdown]: I see what you mean. Simplified it a bit to make it more readable.

File: source/_posts/2023-06-07-release-20236.markdown [hunk 11]
```diff
@@ -0,0 +1,870 @@
...
+
+I am excited about so many things being shipped in this release that it is
+hard to pick a favorite 😅 This release has it all.
+
+Being able to directly use and back up my Home Assistant configuration to my
+network storage is absolutely incredible! But I also really enjoy the new and
+clean integrations dashboard and pages.
+
+The feature that greatly improved the Home Approval Factor in my household
+is definitely the addition of the new favorite colors they can quickly pick
```
Comment (raman325) [inline source/_posts/2023-06-07-release-20236.markdown]: `so they can quickly be picked`?

File: source/_posts/2023-06-07-release-20236.markdown [hunk 12]
```diff
@@ -0,0 +1,870 @@
...
+you are undoubtedly familiar with the annoying scrolling in that little card. Or,
+big error messages stretching those cards out, messing up the whole page.
+
+This release brings in a whole new integrations dashboard that addresses many
+of these issues, aiming to provide a cleaner and more consistent experience.
+
+<img class="no-shadow" src='/images/blog/2023-06/integrations-dashboard.png' alt='Screenshot showing the look and feel of the new integrations dashboard.'>
+
+Device base integrations, like ESPHome, are combined: no more scrolling.
+A failing integration does not stretch the layout and no longer makes the
```
Comment (raman325) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
A failing integration no longer stretches the layout or makes the
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 13]
```diff
@@ -0,0 +1,870 @@
...
+[@piitaya]: https://github.com/piitaya
+
+## Related information
+
+Which automation is using that blueprint? Well, Home Assistant can now tell you
+that! On the {% my blueprints title="**Settings** > **Automations & Scenes** > **Blueprints**" %}
+page, select the three dotted menu on any blueprint to discover which
+automations are using that blueprint.
+
+Not only that, if you accidentally remove a blueprint that is still in use,
```
Comment (raman325) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
Not only that, if you accidentally try to remove a blueprint that is still in use,
```

File: source/_posts/2023-06-07-release-20236.markdown [hunk 14]
```diff
@@ -0,0 +1,870 @@
...
+Home Assistant will stop you from removing it and tell you which automations
+still rely on it.
+
+Hold on; there is more related! The related items in the entity information
+dialog got improved as well. Told old list of links is gone, and it now shows
```
Comment (raman325) [inline source/_posts/2023-06-07-release-20236.markdown]:
```suggestion
dialog were improved as well. Told old list of links is gone, and it now shows
```

**Review by raman325 — COMMENTED** [review]
couple of grammar things

File: source/_posts/2023-06-07-release-20236.markdown (hunk 14 above)
Comment (frenck) [reply source/_posts/2023-06-07-release-20236.markdown]: ~~Seems like you reviewed on older version :)~~

File: source/_posts/2023-06-07-release-20236.markdown (hunk 11 above)
Comment (frenck) [reply source/_posts/2023-06-07-release-20236.markdown]: That part was removed (GitHub was having issues).

**Pull request merged**